src/palette/internal/palettecompat.cpp

```diff
@@ -98,7 +94,22 @@ void PaletteCompat::migrateOldPaletteItemIfNeeded(ElementPtr& element, Score* pa
         newPedal->setEndText(newPedal->propertyDefault(Pid::END_TEXT).value<String>());
 
         element.reset(newPedal);
-        return;
+    }
+
+    element->styleChanged();
+
+    // from 4.4.3 markers have a font size which is not the default, so this must be reset after calling style changed
+    if (item->isMarker() && toMarker(item)->size() == 11) {

```

Comment:
I'm not sure I understand what is going on here. Using 11 and 18 like this looks a bit magic numbers to me (same for the TRead code above) as I don't know the context. They should be in a variable (or at least have an explanatory comment).
Speaking of comment: what do you mean that markers have a font size which is not the default? Shouldn't an element inserted into the score from the default palettes have default settings by definition?